lightning/webelement.py

```diff
@@ -0,0 +1,203 @@
+        self._elements_instance = elements_instance
+        self._element_id = element_id
+        self._session_id = session_id
+
+    def accessibility(self) -> Accessibility:
+        _accessibility = Accessibility(elements_instance=self._elements_instance,
+                                       session_id=self.session_id,
+                                       element_id=self.element_id)
+        return _accessibility
+
+    @property
+    def element_id(self) -> str:
+        return self._element_id
+
+    @property
+    def session_id(self) -> str:
+        return self._session_id
+
+    def click(self) -> WebElement:
+        _web_element = self._elements_instance.element_click(session_id=self.session_id,
+                                                             element_id=self.element_id,
+                                                             body=EmptyRequest())
+        return self
+
+    def clear(self) -> WebElement:
+        _web_element = self._elements_instance.element_clear(session_id=self.session_id,
+                                                             element_id=self.element_id,
+                                                             body=EmptyRequest())
+        return self
+
+    def find_all(self, locator: Locator) -> list[WebElement]:
+        _body = FindElementRequest(using=locator.strategy, value=locator.expression)
+        _response: list[FindElementResponseValue] = self._elements_instance.find_elements(session_id=self.session_id,
+                                                                                          body=_body).value
+        # ? raise smth if len(_response) == 0
+        _web_elements = list(map(lambda e: WebElement(elements_instance=self._elements_instance,
+                                                      element_id=e.element_6066_11e4_a52e_4f735466cecf,
+                                                      session_id=self.session_id), _response))
+        return _web_elements
+
+    def find_first(self, locator: Locator) -> WebElement:
+        try:
+            _body = FindElementRequest(using=locator.strategy, value=locator.expression)
+            _response: FindElementResponseValue = self._elements_instance.find_element(session_id=self.session_id,
+                                                                                       body=_body).value
+            _web_element = WebElement(elements_instance=self._elements_instance,
+                                      element_id=_response.element_6066_11e4_a52e_4f735466cecf,
+                                      session_id=self.session_id)
+            return _web_element
+        except Exception as e:
+            raise WebDriverException() from e
+
+    @property
+    def selected(self) -> bool:
+        _response = self._elements_instance.is_element_selected(session_id=self.session_id, element_id=self.element_id)
```

Comment:
Probably can be shortened to one line:
```
return self._elements_instance.is_element_selected(session_id=self.session_id, element_id=self.element_id).value
```

Reply:
fixed in 990528b5d10e02f0199fddd9a570dc33e95fa248